0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -110,387 +110,380 @@ informative:
 
-### Multi-Level
-Crypto-conditions elegantly support weighted multi-signatures and multi-level signatures. A threshold condition has a number of weighted subconditions, and a target threshold. Each subcondition can be a signature or another threshold condition. This provides flexibility in forming complex conditions.
+## Multi-Level
+Crypto-conditions elegantly support weighted multi-signatures and multi-level signatures. A threshold condition has a number of subconditions, and a target threshold. Each subcondition can be a signature or another threshold condition. This provides flexibility in forming complex conditions.
 
-For example, consider a threshold condition that consists of two subconditions, one each from Agnes and Bruce. Agnes's condition can be a signature condition while Bruce's condition is a threshold condition, requiring both Claude and Dan to sign for him.
+For example, consider a threshold condition that consists of two subconditions, one each from Wayne and Alf. Alf's condition can be a signature condition while Wayne's condition is a threshold condition, requiring both Claude and Dan to sign for him.
 
-Weighted signatures allow more complex relationships than simple M-of-N signing. For example, a weighted condition can support an arrangement of subconditions such as, "Either Ron, Adi, and Leonard must approve; or Clifford must approve."
+Multi-level signatures allow more complex relationships than simple M-of-N signing. For example, a weighted condition can support an arrangement of subconditions such as, "Either Ron, Mac, and Ped must approve; or Smithers must approve."
 
 
+## Crypto-conditions as a signature scheme
 
-# Format
+Crypto-conditions is a signature scheme for compound signatures which has similar properties to most other signature schemes, such as:
+  1. Validation of the signature (the fulfillment) is done using a public key (the condition) and a message as input
+  2. The same public key can be used to validate multiple different signatures, each against a different message
+  3. It is not possible to derive the signature from the public key
+  
+However, the scheme also has a number of features that make it unique such as:
+  1. It is possible to derive the same public key from any valid signature without the message
+  2. It is possible for the same public key and message to be used to validate multiple signatures. For example, the fulfillment of an m-of-n condition will be different for each combination of n signatures.
+  4. Composite signatures use one or more other signatures as components allowing for recursive signature validation logic to be defined.  
+  3. A valid signature can be produced using different combinations of private keys if the structure of the compound signature requires only specific combinations of internal signatures to be valid  (m of n signature scheme).
+  
+## Crypto-conditions as a trigger in distributed systems
```

Comment:
No, the trigger use case is not the core problem crypto-conditions are solving. Quite the opposite, it's the motivating use case, but that's mostly a historical accident. A much more typical use case for crypto-conditions would be multi-sig authentication.

For instance, when rippled added support for multi-sig, crypto-conditions would have been a very elegant way to express the keys. We just didn't have them back then, so we came up with a custom way of doing it. But that was a much more typical use case than SusPay.